_posts/2020-12-10-share-your-knowledge.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: epic
+title: "Share You a Knowledge for Great Good!"
```

Comment:
Non-blocking: Heads up that I was hung up for a bit trying to understand the title. I get it now but it took me a little bit to figure out that it was intentionally poor grammar.

Reply:
I have a tendency to be clever in my titles and it gets me in trouble. I'll rephrase 👍